ergo/distributions/histogram.py

```diff
@@ -77,8 +77,59 @@ def initialize_optimizable_params(fixed_params):
         num_bins = fixed_params.get("num_bins", 100)
         return onp.full(num_bins, -float(num_bins))
 
-    def normalize(self):
-        return HistogramDist(self.logps, scale=Scale(0, 1))
+    def normalize(self, true_scale: Scale = None):
```

Comment:
Feels like this should be split into two functions:

1. Expand the scale and add empty bins
2. Normalize

Is there a reason to merge these?

Reply:
nope!